src/cpuinfo_aarch64.c

```diff
@@ -119,23 +122,8 @@ int GetAarch64FeaturesEnumValue(const Aarch64Features* features,
 }
 
 const char* GetAarch64FeaturesEnumName(Aarch64FeaturesEnum value) {
-  switch (value) {
-    case AARCH64_FP:
-      return "fp";
-    case AARCH64_ASIMD:
-      return "asimd";
-    case AARCH64_AES:
-      return "aes";
-    case AARCH64_PMULL:
-      return "pmull";
-    case AARCH64_SHA1:
-      return "sha1";
-    case AARCH64_SHA2:
-      return "sha2";
-    case AARCH64_CRC32:
-      return "crc32";
-    case AARCH64_LAST_:
-      break;
-  }
-  return "unknown feature";
+  if(value >= kConfigsSize)
+    return "unknown feature";
+  else
```

Comment:
Ha also can you remove the `else` here and below?
Sorry I should have mentioned it in the previous review...

Reply:
fixed